parts/k8s/cloud-init/artifacts/pam-d-su

```diff
@@ -0,0 +1,61 @@
+#
+# The PAM configuration file for the Shadow `su' service
+#
+
+# This allows root to su without passwords (normal operation)
+auth       sufficient pam_rootok.so
+
+# Uncomment this to force users to be a member of group root

```

Comment:
Can we make this file "lighter" by removing a lot of these commented out lines?

Reply:
I have a slight preference for keeping things as-is, as that's the default shipped w/ Ubuntu 16.04.

Reply:
It would not be hard to write a preprocessor function that stripped out comments before the files landed on the VHD or disk. I think we should do that—then we could apply it declaratively to config files. (And after that, minify the shell scripts...)